yml/apdb.yaml

```diff
@@ -2075,6 +2124,96 @@ tables:
   - name: bboxSize
     "@id": "#DiaSource.bboxSize"
     datatype: long
+  - name: pixelFlags
+    "@id": "#DiaSource.pixelFlags"
+    datatype: boolean
+    description: General failure flag, set if anything went wrong
+    fits:tunit:
+  - name: pixelFlags_bad
```

Comment:
I don't see `pixelFlags_badCenter`: I think we want that too.

Reply:
Deferring additions to DM-44151.